resources/compass/charts/connector/templates/service-role-binding.yaml

```diff
@@ -0,0 +1,25 @@
+{{ if .Values.istio.rbac.enabled  }}
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: {{ template "fullname" . }}-internal
+  namespace: {{ .Release.Namespace }}
+spec:
+  rules:
+    - services: ["{{ template "fullname" . }}-internal.compass-system.svc.cluster.local"]
+      paths: ["*"]
+      methods: ["*"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRoleBinding
+metadata:
+  name: {{ template "fullname" . }}-internal
+  namespace: {{ .Release.Namespace }}
+spec:
+  subjects:
+{{ tpl (toYaml .Values.istio.rbac.connectorInternalApi.subjects | indent 4) . }}
```

Comment:
Is there a reason for the usage of `tpl` function here? 
I don't see any templates used in [values for subjects](https://github.com/kyma-project/kyma/pull/6086/files#diff-3279d290b32336fd91b3d36f4999505dR31).

Reply:
Good catch! Thanks 😀 I'll leave it as it is for now because it could be needed some day